common/browser_utilities.py

```diff
@@ -38,7 +62,10 @@ def wait_for_element_by_id(element_id, timeout, retry):
     Returns:
         boolean: True if element found else False
     """
-    return wait_for(_element_by_id_on_page, timeout, retry, element_id)
+    ret_val = wait_for(_element_by_id_on_page, timeout, retry, element_id)
+    assert ret_val
+
+    return ret_val
 
 
 def wait_for(fn, timeout, retry_after, *argv):
```

Comment:
"No" is a perfectly valid repsonse, but do you think we could do with having some defaults for this function?